test/mp_unit/proxy_channel_tests.cu

```diff
@@ -234,17 +240,21 @@ void ProxyChannelOneToOneTest::testPingPongPerf(bool useIbOnly, bool waitWithPol
   proxyService->stopProxy();
 }
 
-TEST_F(ProxyChannelOneToOneTest, PingPong) { testPingPong(false, false); }
+TEST_F(ProxyChannelOneToOneTest, PingPong) { testPingPong(true, true, false, false); }
```

Comment:
Hmm... `true, true, false, false` is not easy to understand. Can we use a struct for this? Such as 
```cpp
struct Params {
    bool useIb = true;  // default values
    bool useIPC = false;
};
Params p{.useIb = false, .useIPC = true}; 
```